style.css

```diff
@@ -989,6 +1002,7 @@ a:hover .nav-title,
 
 .navigation-top .wrap {
 	padding: 0;
+	max-width: -webkit-calc( 1000px + 2.5em);
```

Comment:
While this PR is just related to prefixing, seems like we have some preexisting CSS guideline issues which are also being added to the prefixed values.

Extra space after opening parenthesis.
```css
max-width: -webkit-calc( 1000px + 2.5em);
```
should be 
```css
max-width: -webkit-calc(1000px + 2.5em);
```